src/js/view/view.js

```diff
@@ -804,6 +804,9 @@ function View(_api, _model) {
                 if (_captionsRenderer) {
                     _captionsRenderer.hide();
                 }
+                if (_preview && !_model.get('isFloating')) {
```

Comment:
youu're checking floating in this function as well - only need to do it in one spot - i'd suggest in `enableZoomThumbnail`